kolibri/plugins/user/assets/src/views/profile-page/index.vue

```diff
@@ -26,34 +26,38 @@
 
     <form @submit.prevent="submitEdits">
 
+      <h3>{{ $tr('name') }}</h3>
```

Comment:
This is going to conflict with #2190. Incorporate the patch in that PR here and close #2190.